ci/scripts/install_numba.sh

```diff
@@ -48,10 +48,21 @@ fi
 
 numba_cuda=$2
 
+if [ "$#" -eq 3 ]; then
+  cuda=$3
+else
+  # Default to CUDA 11
+  cuda=11
+fi
```

Comment:
I'm curious, is it possible to query the system to get the CUDA version instead of having to pass it explicitly?